src/EnliteMonolog/Service/MonologServiceFactory.php

```diff
@@ -57,6 +58,10 @@ public function createLogger($container, MonologOptions $options)
             $logger->pushProcessor($this->createProcessor($container, $processor));
         }
 
+        if (class_exists('\Monolog\ErrorHandler') && $options->isErrorHandler()) {
+            ErrorHandler::register($logger);
```

Comment:
I think that it's to late to register error handler. What about bootstrap or init event?